recipes/boost-multi/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "0.80.1":
+    url: https://gitlab.com/correaa/boost-multi/-/archive/v0.80.1/boost-multi-v0.80.1.tar.gz
+    sha256: 7d1ebee123783e0fadfb58a0f55309148d63295fcb4b9c5442d851e3578cad1e
```

```suggestion
  "0.83.0":
    url: "https://gitlab.com/correaa/boost-multi/-/archive/v0.83.0/boost-multi-v0.83.0.tar.gz"
    sha256: "52324a01e25fd66a32c8bf88f41940ba6364fd218cfab53b7ec5c63115dea1bf"
```

I would recommend updating to 0.83.0, so the LICENSE file will be available.